openshift/templates/cachito-volumes.yaml

```diff
@@ -0,0 +1,71 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: cachito-volumes
+  description: >
+    Persistent volumes for cachito archives.
+
+    The reclaim policy for this volume is Retain, meaning the data is never
+    removed (not by openshift, anyway).
+
+parameters:
+  - name: CACHITO_ARCHIVES_DIR
+    description: Local directory for cachito archives
+    required: true
+  - name: CACHITO_ATHENS_DIR
+    description: Local directory for athens storage
+    required: true
+
+labels:
+  volume: cachito-volume
+
+objects:
+  - kind: PersistentVolume
+    apiVersion: v1
+    metadata:
+      name: cachito-archives
+    spec:
+      hostPath:
+        path: ${CACHITO_ARCHIVES_DIR}
+      accessModes:
+        - ReadWriteMany
+      capacity:
+        storage: 100Gi
+      persistentVolumeReclaimPolicy: Retain
+
+  - kind: PersistentVolumeClaim
+    apiVersion: v1
+    metadata:
+      name: cachito-archives
+    spec:
+      volumeName: cachito-archives
+      accessModes:
+        - ReadWriteMany
+      resources:
+        requests:
+          storage: 100Gi
+
+  - kind: PersistentVolume
+    apiVersion: v1
+    metadata:
+      name: cachito-athens-storage
+    spec:
+      hostPath:
+        path: ${CACHITO_ATHENS_DIR}
+      accessModes:
+        - ReadWriteOnce
+      capacity:
+        storage: 100Gi
+      persistentVolumeReclaimPolicy: Retain
+
+  - kind: PersistentVolumeClaim
+    apiVersion: v1
+    metadata:
+      name: cachito-athens-storage
+    spec:
+      volumeName: cachito-athens-storage
+      accessModes:
+        - ReadWriteOnce
+      resources:
+        requests:
+          storage: 100Gi
```

Comment:
This seems rather high for a dev environment

Reply:
We are just not limiting the resources for those volumes. I did it this way to keep it consistent to the other PVs, such as the koji and registry ones